doc/user/pim.rst

```diff
@@ -319,7 +319,7 @@ MSDP can be setup in different ways:
 
 Commands available for MSDP:
 
-.. clicmd:: ip msdp timers (2-600) (3-600) [(1-600)]
+.. clicmd:: ip msdp timers (1-65535) (1-65535) [(1-65535)]
 
    Configure global MSDP timers.
 
```

Comment:
The comments here needs updating: you are right: local `keep-alive` interval can be lower than local `hold-time` as long as the remote end has a higher `hold-time`.

Reply:
updated doc.